### PR TITLE
Assorted Bugfixes (migrations, multichain, condition schema)

### PR DESCRIPTION
**Pull request by KPrasch**
**Type of PR:**
- Bugfix

**Required reviews:** 
- 1

**What this does:**
- Fixes an `AttributeError` upon condition evaluation (specifically during POA chain setup)
- Corrects incorrectly named configuration migration field (`condition_providers` -> `condition_provider_uris`)
- Adds missing `conditionType` field in `nucypher/examples`

**Notes**
- This PR is tagged `dkg-dev-11` and deployed to `lynx`

### CODE REVIEW

File: examples/tdec/simple_tdec_local.py
Comment (piotr-roslaniec) [inline examples/tdec/simple_tdec_local.py:1]: Aren't these run on CI?

**Review by piotr-roslaniec — APPROVED** [review]

File: examples/tdec/simple_tdec_local.py
Comment (KPrasch) [reply examples/tdec/simple_tdec_local.py:1]: As far as I can tell, no, but your point is taken!

**Review by manumonti — APPROVED** [review]
LGTM! ✌️

File: nucypher/config/migrations/configuration_v4_to_v6.py [hunk 1]
```diff
@@ -4,19 +4,18 @@
...
-    del config["federated_only"]  # deprecated
-    del config["checksum_address"]
-
-    # Multichain support
+def __migration(config: Dict) -> Dict:
     eth_provider = config["eth_provider_uri"]
     eth_chain_id = NetworksInventory.get_ethereum_chain_id(config["domain"])
     polygon_provider = config["payment_provider"]
     polygon_chain_id = NetworksInventory.get_polygon_chain_id(config["payment_network"])
-    config["condition_providers"] = {
```
Comment (derekpierre) [inline nucypher/config/migrations/configuration_v4_to_v6.py:16]: Should "condition_providers" be deprecated then?

i.e.
```python
del config["condition_providers"]
```

File: nucypher/config/migrations/configuration_v5_to_v6.py [hunk 2]
```diff
@@ -4,15 +4,18 @@
 from nucypher.config.migrations.common import perform_migration
 
 
-def __migration(config: Dict) -> None:
+def __migration(config: Dict) -> Dict:
     eth_provider = config["eth_provider_uri"]
     eth_chain_id = NetworksInventory.get_ethereum_chain_id(config["domain"])
     polygon_provider = config["payment_provider"]
     polygon_chain_id = NetworksInventory.get_polygon_chain_id(config["payment_network"])
-    config["condition_providers"] = {
```
Comment (derekpierre) [inline nucypher/config/migrations/configuration_v5_to_v6.py:12]: Same comment about deprecation as above.

**Pull request merged**